app/uk/gov/hmrc/nisp/views/includes/helpers/error_overview.scala.html

```diff
@@ -0,0 +1,16 @@
+@(form: Form[_])
+
+@if(form.hasErrors) {
+        <div id="error-summary-display" class="flash error-summary error-summary--show " role="group" aria-labelledby="error-summary-heading" tabindex="-1">
+            <h2 id="error-summary-heading" class="h3-heading">@Messages("nisp.validation.summary.heading")</h2>
+            <ul class="js-error-summary-messages">
+                @form.errors.map { error =>
+                <li role="tooltip">
+                    <a href='#@error.key' id='@error.key-error-summary' data-focuses='@error.key'>
+                        @if(error.key=="email") {@Messages("nisp.validation.summary.email") }else{ @Messages(error.message, error.args : _*)}
```

Comment:
I mean you've already wrote `nisp.validation.summary.error.max = Maximum length is 1,200` which contains the full message.